releases/development/master/extra/barclamp_install.rb

```diff
@@ -109,15 +109,25 @@ def rm_tmpdir(tmpdir)
     puts "Barclamp at #{bc} has no name, skipping"
     next
   end
+  version = barclamp["barclamp"]["version"].to_i rescue 0
   order = 9999
```

Comment:
Then the result would be different. Imagine `["crowbar"]["order"] = nil`. The old code would return 9999 (it is checked in the condition that ["order"] value is not nil), the new code would return 0, because `nil` gets converted to 0 by `.to_i`. 

For the version, it does not matter, because 0 is a good default.